### ISSUE TITLE
Remove requirejs-dplugins/css.js dojo dependency

### ISSUE THREAD

**Issue by pruzand** [ibm-js/requirejs-dplugins#4]
Today the requirejs-dplugins/css plugin depends on dojo/Deferred.
While it was not a real issue when the plugin was part of the delite project, this dependency is more critical now that it has been moved to requirejs-dplugins. Indeed, if a project depends on it, it will bring all dojo with it. This is the case if a project depends on ecma402 for example: in this case you end up with a dependency on dojo even if you don't use it, just because the requirejs-dplugins/css requires Deferred.
IMO it would be great if the css plugin would depend on another promises implementation.
